ath79: fix USB power on TL-WR810N v1

Torxgewinde opened the pull request:
Before: Kernel reported "usb_vbus: disabling" and the USB was not providing power
After: USB power is switched on, peripheral is powered from the device

Signed-off-by: Tom Stöveken <tom@naaa.de>

Torxgewinde committed:
Switch USB power on

Before: Kernel reported "usb_vbus: disabling" and the USB was not providing power
After: USB power is switched on, peripheral is powered from the device.

adschm commented:
Hi, please add a more descriptive title to the commit, e.g. "ath79: fix USB power on TL-WR810N v1".

Apart from that, please add a Signed-off-by line with your real name and e-mail address to your commit message.

Torxgewinde commented:
Hi Adrian, is that edit I made in here acceptable? I am not too experienced with git and still picking up the details.

Torxgewinde committed:
ath79: fix USB power on TL-WR810N v1

Before: Kernel reported "usb_vbus: disabling" and the USB was not providing power
After: USB power is switched on, peripheral is powered from the device

Signed-off-by: Tom Stöveken <tom@naaa.de>

Torxgewinde commented:
Hi, (I think) editorial changes where applied...

adschm commented:
Normally, you would squash the two commit together into one. I will do that for you in this case.

adschm commented:
Thanks, will be merged later today.
